docs/tutorials/google/compare_to_calibration.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<p>We have only plotted the exponential fits and not the survival probability vs cycle depth below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Outdated, Loschmidt echo removed. (The exponential fits were the thin black lines, and the survival probabilities were the dotted lines.)</p>